expression/builtin_other_vec_generated.go

```diff
@@ -188,7 +188,7 @@ func (b *builtinInDecimalSig) vecEvalInt(input *chunk.Chunk, result *chunk.Colum
 			arg0 := args0[i]
 			arg1 := args1[i]
 			compareResult = 1
```

Comment:
Seems we don't need this variable?

Reply:
This logic is generated from our template and this variable makes it easier for us to write template code.

Reply:
Got